_posts/2016/2016-07-04-ep6-offsite.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Team offsite meeting: workshops, focus, and fun"
+date: "2016-07-04 13:37:42"
+image: blog-header/ep6offsite2016_back.jpg
```

Comment:
I processed this image for usage in the blog. Please use that one:
![ep6offsite2016_back](https://cloud.githubusercontent.com/assets/10460958/16555240/f60e39f6-41d3-11e6-83a8-9d9e4ce1ac9b.jpg)